src/cljs/s3_beam/client.cljs

```diff
@@ -1,10 +1,14 @@
 (ns s3-beam.client
-  (:import (goog Uri))
   (:require-macros [cljs.core.async.macros :refer [go]])
   (:require [cljs.reader :as reader]
             [cljs.core.async :as async :refer [chan put! close! pipeline-async]]
             [goog.dom :as gdom]
-            [goog.net.XhrIo :as xhr]))
+            [goog.events :as events])
+  (:import  (goog Uri)
+            (goog.net XhrIo)
+            goog.net.EventType
+            goog.net.ErrorCode
+            (goog.events EventType)))
```

Comment:
Right, I made the mistake of searching the page with some portions of the code collapsed.
I think the example I gave above should work there might be differences in how you access `ERROR` and `SUCCESS` then. I think probably this should work: `EventType/ERROR`.